packages/cashier/src/modules/deposit-crypto/components/deposit-crypto-wallet-address/deposit-crypto-wallet-address.tsx

```diff
@@ -8,11 +8,12 @@ import { DepositCryptoDisclaimers } from '../deposit-crypto-disclaimers';
 import './deposit-crypto-wallet-address.scss';
 
 const DepositCryptoWalletAddress: React.FC = observer(() => {
-    const { ui } = useStore();
+    const { ui, client } = useStore();
     const { is_mobile } = ui;
+    const { is_switching } = client;
     const { data: deposit_crypto_address, isLoading, error, resend } = useDepositCryptoAddress();
 
-    if (isLoading) return <Loading is_fullscreen={false} />;
+    if (isLoading || is_switching) return <Loading is_fullscreen={false} />;
```

Comment:
here we added `is_switching` to avoid blinking of QR code when we try to open wallet-modal using deposit shortcut on main TradersHub page for different crypto wallets

Reply:
@sergei-deriv Would it be possible to do this in the wallet code somehow?